.github/workflows/deploy-pr.yml

```
@@ -0,0 +1,29 @@
+name: deploy website to netlify
+
+on: pull_request
+
+jobs:
+  build-static-page:
+    name: build
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: build image
+        run: docker build -t documentation:latest .
```

Assuming we're not interested in building the archives for pull requests, this can now be:

```suggestion
        run: DOCKER_BUILDKIT=1 docker build --build-arg=ENABLE_ARCHIVES=false --target=deploy-source --output=/output/_site .
```